src/main/java/com/entropyteam/entropay/employees/models/Company.java

```diff
@@ -9,9 +9,14 @@
 import javax.persistence.Table;
 import com.entropyteam.entropay.common.BaseEntity;
 import com.entropyteam.entropay.employees.dtos.CompanyDto;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.NoArgsConstructor;
```

Comment:
For this project we're trying to avoid the use of lombok. @eazaretzky It's not fully compatible with Java 17, is it?

Reply:
Looking at this I haven't found any compatibility issues regarding Lombok and Java17. Could you please confirm whether we should modify this or not?

Reply:
For the moment, for the moment let's try not to use lombok. But i want to discuss this as a team

Reply:
Deleted lombok